pom.xml

```diff
@@ -99,5 +99,11 @@
 			<artifactId>imgscalr-lib</artifactId>
 			<version>4.2</version>
 		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.12</version>
```

Comment:
Why not JUnit 5? It is stable for more than two years and in my opinion we should start with the actual version.